.travis.yml

```diff
@@ -332,7 +332,7 @@ jobs:
     - if: (type = cron) OR (branch = prerelease_test)
       stage: test
       name: "CodeCov 10.12"
-      env: PG_VERSION=10.12 OTHER_CMAKE_FLAGS="-DCMAKE_BUILD_TYPE=Debug  -DTS_COVERAGE_ONLY=TRUE -DCMAKE_C_FLAGS='-O0 -coverage -fprofile-arcs -ftest-coverage -DNDEBUG'" CODECOV_FLAGS="-F cron"
+      env: PG_VERSION=10.12 OTHER_CMAKE_FLAGS="-DCMAKE_BUILD_TYPE=Debug -DCMAKE_C_FLAGS='-O0 -coverage -fprofile-arcs -ftest-coverage -DNDEBUG'" CODECOV_FLAGS="-F cron"
```

Comment:
It might be good to explicitly turn off assertions on the command line, to make it clear that they are disabled.

Reply:
This is not sufficient as we might pick up the setting from the PostgreSQL configuration in pg_config.h. 

It is much better to print the status of assertions in the CMake configuration run, which this PR does, as that is the ultimate source of truth of whether this setting is on or off.